src/Cake.Curl.Tests/CurlUploadFileTests.cs

```diff
@@ -477,6 +478,76 @@ public void Should_Not_Set_The_RetryConnRefused_Option_As_Argument()
                 // Then
                 Assert.DoesNotContain("--retry-connrefused", result.Args);
             }
+
+            [Theory]
+            [MemberData(nameof(CurlTimeSpanTestData.TimeSpanData), MemberType = typeof(CurlTimeSpanTestData))]
+            public void Should_Set_The_MaxTime_Option_As_Argument(TimeSpan maxTime)
```

Comment:
I think this test could be simplified by passing the number of seconds as a parameter and create the `TimeSpan` during the _arrange_ phase:

```csharp
[Theory]
[InlineData(0.5)]
[InlineData(60)]
[InlineData(2.99)]
public void Should_Set_The_MaxTime_Option_As_Argument(double seconds)
{
    // Given
    var maxTime = TimeSpan.FromSeconds(seconds);
    ...
}
```